locales/en/app.ftl

```diff
@@ -514,6 +514,12 @@ want-to-add = Want to add another email?
 # the form to add an additional email to Firefox Monitor.
 verify-the-link = Verify the link sent to { $userEmail } to add it to {-product-name}.
 
+# These are part of a confirmation page that appears after a user has verified
```

Comment:
Fluent has section comments, let's use those.

```
## Used in confirmation page that appears after a user has verified
## an additional email to Firefox Monitor.
```

Lave an empty line after this comment and before the strings. Then *reset* the section comment after the strings with just `##`.